3ds/update_presence.go

```diff
@@ -1,13 +1,24 @@
 package friends_3ds
 
 import (
+	database_3ds "github.com/PretendoNetwork/friends-secure/database/3ds"
 	"github.com/PretendoNetwork/friends-secure/globals"
 	nex "github.com/PretendoNetwork/nex-go"
 	nexproto "github.com/PretendoNetwork/nex-protocols-go"
 )
 
-func UpdatePresence(err error, client *nex.Client, callID uint32, presence *nexproto.NintendoPresence, unknown bool) {
-	// TODO: Do something with this
+func UpdatePresence(err error, client *nex.Client, callID uint32, presence *nexproto.NintendoPresence, showGame bool) {
+	currentPresence := presence
+
+	// Send an entirely empty status, with every flag set to update
```

Comment:
Do we know what the other flags are?

Reply:
No clue, only FF FF FF FF, which is all flags. No other ones seem to be useful, as this is only for removing all of the data to hide what the player is doing.

Reply:
Okay. I was asking because if other flags were known then it would likely be better to also use an enum or global variable for those as well. But if not then hard coding it should be fine